packages/jest-haste-map/src/HasteFS.js

```diff
@@ -78,7 +78,7 @@ export default class HasteFS {
     const files = new Set();
     for (const file of this.getAbsoluteFileIterator()) {
       const filePath = root ? fastPath.relative(root, file) : file;
-      if (micromatch([filePath], globs).length) {
+      if (micromatch.some(filePath, globs)) {
```

Comment:
does `fastPath.relative` return forward slashes on Windows?

Reply:
no it uses `path.relative`, `path.resolve` and `path.sep`